docker/stubs/flac/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ouspg/libfuzzer-base
+
+MAINTAINER https://github.com/ouspg/libfuzzerification
+ENV TARGET 'flac'
+
+#Install dependencies and fetch the source.
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y libflac-dev pkg-config libtool-bin && apt-get build-dep libflac-dev -y
+RUN cd /src && git clone https://git.xiph.org/flac.git
+ADD flac_fuzzer.c /src/$TARGET/
+
+RUN mkdir -p /work/$TARGET/
```

Comment:
Now that we are not bulding our own clang+llvm and libFuzzer is used as an actual lib, /work should be unused.